json2csv.py

```diff
@@ -0,0 +1,66 @@
+import click
+import json
+import csv
+import logging
+import re
+
+
+logger = logging.getLogger(__name__)
+
+DATASET_KEY = 'nationalrat'
+SOURCE_ID = 'open_parliament_at'
+
+WANTED_FIELDS = ['first_name', 'last_name', 'salutation', 'title', 'political_affiliation',
+                 'state', 'occupation', 'place_of_birth', 'date_of_birth']
+
+party_extract_re = re.compile(r'^.* \((?P<party>\w+)\)$')
+
+@click.command()
+@click.argument('input', type=click.File('rb'))
+@click.argument('output', type=click.File('w'))
```

Comment:
Please let those default to stdin/stdout (respectively). This makes the script far more versatile in it’s usage. Also you can turn the arguments into options this way.